package.json

```diff
@@ -2,6 +2,11 @@
   "scripts": {
     "test": "xo"
   },
+  "dependencies": {
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-webstore-upload": "^0.8.10"
+  },
```

Comment:
@paulmolluzzo @sindresorhus I still don't get why we would be using `dependencies` for that? You don't need `grunt` or `grunt-cli` or `grunt-webstore-upload` to use the extension, a developer would need them to publish to the webstore. The same way when there are dependencies only for tests we put them in `devDependencies` **and install them on Travis C*I* we should add these as `devDependencies` and install them on Travis CI. It's just that Travis CI is doing the publishing, not a developer manually.